### ISSUE TITLE
Error in  Seq2Seq_solutions.ipynb

### ISSUE THREAD

**Issue by dixiematt8** [Kyubyong/tensorflow-exercises#6]
This line has error

`loss = tf.nn.sparse_softmax_cross_entropy_with_logits(logits=logits, labels=y)
`

It should be

`loss=tf.nn.softmax_cross_entropy_with_logits(logits=logits,labels=dec_inputs)
`
